tutorials/logical_audio_sensor.md

```diff
@@ -0,0 +1,195 @@
+\page logicalaudiosensor Logical Audio Sensor
+
+This tutorial will explain how to use the `LogicalAudioSensor` system plugin in Ignition Gazebo.
+
+The logical audio sensor plugin allows for the usage of logical audio sources and microphones in a simulation environment.
+At the end of each simulation step, microphones check if audio was detected by any of the sources in the world.
+The logical audio plugin does not play actual audio to a device like speakers, but rather simulates audio being played in environments to see if audio could theoretically be heard at a certain location or not.
+
+## Setup
+
+Let's take a look at [logical_audio_sensor_plugin.sdf](https://github.com/ignitionrobotics/ign-gazebo/blob/1eb4253c6283e711e983be2b398bc7c0411c9408/examples/worlds/logical_audio_sensor_plugin.sdf), which defines a simulation world with 4 models (in this case, boxes) that have an audio object attached to them.
+This world attaches logical audio sources to the `red_box` and `blue_box` models, and attaches logical microphones to the `green_box` and `yellow_box` models.
+
+Let's take a look at the SDF relevant to the source for `red_box` to understand how to define a logical audio source in SDF:
+
+```xml
+      <plugin filename="ignition-gazebo-logicalaudiosensorplugin-system" name="ignition::gazebo::systems::LogicalAudioSensorPlugin">
+        <source>
+          <id>1</id>
+          <pose>.5 0 0 0 0 0</pose>
+          <attenuation_function>linear</attenuation_function>
+          <attenuation_shape>sphere</attenuation_shape>
+          <inner_radius>1.0</inner_radius>
+          <falloff_distance>5.0</falloff_distance>
+          <volume_level>.8</volume_level>
+          <playing>true</playing>
+          <play_duration>10</play_duration>
+        </source>
+      </plugin>
+```
+
+As we can see, we use a `<source>` tag to define the source.
+An explanation of all of the tags can be found in the [plugin documentation](https://github.com/ignitionrobotics/ign-gazebo/blob/1eb4253c6283e711e983be2b398bc7c0411c9408/src/systems/logical_audio_sensor_plugin/LogicalAudioSensorPlugin.hh#L45-L86), but there are a few important things to point out:
+* `<id>` is used to identify this source when operating on it via services (services will be discussed later).
+Since a model can have multiple sources and microphones attached to it, each source attached to a particular model must have a unique ID.
+This means that no other sources attached to `red_box` can have an ID of 1, but sources attached to other models can have an ID of 1 (assuming that other models don't already have a source with an ID of 1 attached to it).
+* The source's pose is defined relative to the model pose it's attached to - so, in this case, since the `red_box` pose is `(0, 0, 0.5, 0, 0, 0)` relative to the world, and the source pose is `(0.5, 0, 0, 0, 0, 0)` relative to the `red_box` pose, the source's pose with respect to the world is `(0.5, 0, 0.5, 0, 0, 0)`.
```

Comment:
```suggestion
* The source's pose is defined relative to the model's pose it's attached to. In this case, since the `red_box` pose is `(0, 0, 0.5, 0, 0, 0)` relative to the world, and the source pose is `(0.5, 0, 0, 0, 0, 0)` relative to the `red_box` pose, the source's pose with respect to the world is `(0.5, 0, 0.5, 0, 0, 0)`.
```

Reply:
Fixed in https://github.com/ignitionrobotics/ign-gazebo/pull/401/commits/bb50ab5e21d6fec3c54a1e1b7fdd6623fc285908